src/index.ts

```diff
@@ -169,4 +169,7 @@ export type {TruncateProps} from './Truncate'
 export {default as UnderlineNav} from './UnderlineNav'
 export type {UnderlineNavProps, UnderlineNavLinkProps} from './UnderlineNav'
 
+export {default as Checkbox} from './Checkbox'
```

Comment:
Any ideas how we could guard against engineers rendering a `<Checkbox />` without a corresponding `<label />`?

I have some ideas, but it would be much easier to show over Zoom instead of text and pseudocode. Let me know if you want to chat on Monday.

Reply:
Chatted with @rezrah about this: we're just going to check for an associated label with a linter instead of doing anything in the runtime.